README.md

```diff
@@ -1,280 +1,231 @@
-                       Make sure this is the last argument, cause everything after is
-                       forwarded to Parity.
-
- ROLES
-   The list of available roles is:
-
-   observer
-     - Is the default role
-     - Does only watch for propagated blocks.
-     - Non arguments required at all.
-
-   participant
-     - Connects to an account to being able to create transactions.
-     - Requires the address argument.
-     - Needs the password file and the key-set. (see FILES)
-
-   validator
-     - Connect as authority to the network for validating blocks.
-     - Requires the address argument.
-     - Needs the password file and the key-set. (see FILES)
-
- FILES
-   The configuration folder for Parity takes place at /home/parity/.local/share/io.parity.ethereum.
-   Alternately the shorthand symbolic link at /config can be used.
-   Parity's data base is at /home/parity/.local/share/io.parity.ethereum/chains or available trough /data as well.
-   To provide custom files in addition bind a volume through Docker to the sub-folder called 'custom'.
-   The password file is expected to be placed in the custom configuration folder names 'pass.pwd'.
-   The key-set is expected to to be placed in the custom configuration folder under 'keys/Trustlines/'
-   Besides from using the pre-defined locations, it is possible to define them manually thought the parity
-   arguments. Checkout their documentation to do so.
-```
+Executing the script again is safe: No configuration will be overridden. This allows you to add components not
+configured in earlier runs and will restart all containers.
 
-#### Examples
+### Setup With Docker
 
-Besides the original help, the following sections provide some example instructions how to get started for the different selectable roles.
+A more explicit way of setting up the various components is starting the docker containers manually as described here.
+To keep commands as concise as possible, only the most basic options are provided. You might want to set additional
+ones, e.g., container names or restart policies.
 
-##### Observer
+#### Blockchain Node
 
-For observing the Trustlines blockchain absolutely nothing has to be prepared for the client image. A simple pull and run is all that is needed.
-Since the loss of the block data when the container restarts would lead to a long startup procedure where the clients needs to sync up first, it is
-recommended to use a local volume bound in to save the data outside of the _Docker_ container. Furthermore the node should be able to explore the network and establish
-connections to other peers. Therefore the port `30300` has to be public available and mapped to the containers one.
+First of all, to conveniently allow other containers to connect to the node, create a docker network with
 
 ```sh
-$ mkdir ./database
-$ docker run -ti -v $(pwd)/database:/data -p 30300:30300 trustlines/tlbc-testnet
+$ docker network create network-laika
```

Comment:
Originally I just used the host network for everything to use as few commands as possible. Then I tried to add a network to fix the netstats bug which didn't work. Now I'm not sure what's better anymore.

Reply:
I'd like to hold on with naming changes here until we've sorted them out globally. Same with testnet vs mainnet. We'll have to make another run through at this point anyways..

Reply:
Okay, that sounds reasonable. :+1: